base/context.go

```diff
@@ -113,18 +113,14 @@ func (ctx *Context) GetClientTLSConfig() (*tls.Config, error) {
 	}
 
 	if ctx.Certs != "" {
-		if log.V(1) {
-			log.Infof("setting up TLS from certificates directory: %s", ctx.Certs)
-		}
+		log.Printf("setting up TLS from certificates directory: %s\n", ctx.Certs)
```

Comment:
Is `base` also our only route to the `security` package? We should definitely try to cut off dependencies when we can instead of modifying them.